tests/test_statistical_model.py

```diff
@@ -1,2 +1,13 @@
+from alea import StatisticalModel
+
 def test_gaussian_model():
```

Comment:
I think the name should be updated (and maybe also the filename for this one?)

Reply:
Surely yes.